tests/installation/partitioning_raid.pm

```diff
@@ -191,12 +211,30 @@ sub run {
         assert_screen 'partitioning_raid-partition_role_raw_volume';
         send_key 'alt-n';
         assert_screen 'partition-format';
-        send_key 'alt-d';
+        if (is_storage_ng) {
+            my $not_format_hotkey = 'alt-t';
+        }
+        else {
+            my $not_format_hotkey = 'ctrl-d';
```

Comment:
Same as above, or even better to put it inline, e.g.:
send_key(is_storage_ng() ? alt-t : ctrl-d;